plugins/rich-editor/src/scripts/quill/blots/blocks/CodeBlockBlot.ts

```diff
@@ -19,8 +19,7 @@ export default class CodeBlockBlot extends CodeBlock {
         const domNode = super.create(value) as HTMLElement;
         domNode.setAttribute("spellcheck", false);
         domNode.classList.add("code");
-        domNode.classList.add("isBlock");
-        domNode.classList.add("code-block");
+        domNode.classList.add("codeBlock");
```

Comment:
Since we're dropping the dashes we're going to need to change the `static className` on the blot class to match be `codeBlock`. The same goes for the inline code blot. Otherwise we won't be able to copy/paste them from formatted text.

The same applies to the inline code.

Reply:
Geez, that was a bigger refactor than I expected. I think I got all of it now.